lib/LTP/WhiteList.pm

```diff
@@ -124,12 +131,22 @@ sub is_test_disabled {
 
 sub find_whitelist_testsuite {
     my ($suite) = @_;
+    my $issues;
 
-    my $path = get_var('LTP_KNOWN_ISSUES');
+    my $path = get_var('LTP_KNOWN_ISSUES_LOCAL');
     return undef unless defined($path) and -e $path;
 
     my $content = path($path)->slurp;
-    my $issues = Mojo::JSON::decode_json($content);
+    my $suffix = $path =~ s#.*\.([^.]+)#$1#r;
+    if ($suffix eq 'json') {
+        $issues = Mojo::JSON::decode_json($content);
+    } elsif ($suffix eq 'yaml') {
+        my $yp = YAML::PP->new;
+        $issues = $yp->load_string(decode_utf8 $content);
+    } else {
+        die 'Invalid format';
```

Comment:
I hoped you'd write a bit more descriptive error message instead of copying my example string verbatim.

Reply:
Sorry, fixed.